crates/bevy_core_pipeline/src/prepass/node.rs

```diff
@@ -67,9 +67,9 @@ impl ViewNode for PrepassNode {
                 view: &view_motion_vectors_texture.default_view,
                 resolve_target: None,
                 ops: Operations {
-                    // Blue channel doesn't matter, but set to 1.0 for possible faster clear
+                    // Blue channel doesn't matter, but set to 0.0 for possible faster clear
```

Comment:
More context on what each of the color channels mean here would make this much easier for me to follow as a casual reader.